### ISSUE TITLE
gitGraph: Doesn't seem like branches can start with a number

### ISSUE THREAD

**Issue by axelssonHakan** [mermaid-js/mermaid#3715]
**Describe the bug**
Rendering fails if a branch name/id starts with a number
Tries different escaping but unsuccessfully

(Git supports branches that start with a number)

**To Reproduce**
```
gitGraph
    branch 217_Branch
```

**Expected behaviour**
Should be possible to use branches that start with a number

**Screenshots**
<img width="746" alt="Screenshot 2022-10-24 at 10 36 43" src="https://user-images.githubusercontent.com/5768580/197487889-141e68d0-fc37-41f6-8658-403680f28534.png">

**Code Sample**
```
gitGraph
    branch 217_Branch
```

**Comment by gibson042** [comment]
I think this is a duplicate of #3347 that has already been resolved by #3392 in 0dca4d3393fcbea357522e037f00cee2ea186c0d.

**Comment by axelssonHakan** [comment]
Sorry, I actually tried to search for similar

**Comment by aloisklink** [comment]
> Sorry, I actually tried to search for similar

It's probably because the issue was called `num names` and the PR was called `numeric branch names`, so they never had `number` in the name!

https://github.com/mermaid-js/mermaid/commit/0dca4d3393fcbea357522e037f00cee2ea186c0d should soon be released in Mermaid 9.2.0
